ramls/titles.raml

```diff
@@ -0,0 +1,181 @@
+        displayName: Selection status
+        type: string
+        enum: ["true", "false", "ebsco", "all"]
+        description: Filter to narrow down results based on selection status. Defaults to all.
+        example: "false"
+        required: false
+      filterType:
+        displayName: Resource type
+        type: string
+        enum: ["all", "audiobook", "book", "bookseries", "database", "journal", "newsletter", "newspaper", "proceedings", "report","streamingaudio", "streamingvideo","thesisdissertation",  "website", "unspecified"]
+        description: Filter to narrow down results based on resource type. Defaults to all.
+        example: book
+        required: false
+      filterName:
+        displayName: Query by Title Name
+        type: string
+        description: String to search title name to get a collection of titles
+        example: War and Peace
+        required: false
+      filterIsxn:
+        displayName: Query by ISSN/ISBN
+        type: string
+        description: String to search ISSN and ISBN to get a collection of titles
+        example: 1050-3331
+        required: false
+      filterSubject:
+        displayName: Query by Subject
+        type: string
+        description: String to search subjects to get a collection of titles
+        example: history
+        required: false
+      filterPublisher:
+        displayName: Query by Publisher
+        type: string
+        description: String to search publishers to get a collection of titles
+        example: academic
+        required: false
+    responses:
+      200:
+        body:
+          application/vnd.api+json:
+            description: An array of titles comprising the results of the query.  The array will be paged if its length exceeds the value set with the `count` query param, or the default `count` of 25 in its stead.  The included metadata gives us the total result count outside of the paged context.
+            example: !include examples/titles/titles_get_200_response.json
+  /{title_id}:
+    description: Entity representing a title
+    get:
+      description: Get the title by its unique identifier.
+      queryParameters:
+        include:
+          displayName: Related Resources
+          type: string
+          enum: ["resources"]
+          description: Include related resource obects, each representing a partnering of this title with a package.  Any bogus value, like `include=deciduousTrees`, will be silently ignored.
+          example: "resources"
+          required: false
+      responses:
+        200:
+          body:
+            application/vnd.api+json:
+              description: EPKB data for the title matching {title_id}
+              example: !include examples/titles/titles_titleId_get_200_response.json
+        404:
+          description: Not Found
+          body:
+            application/vnd.api+json:
+              example: !include examples/titles/titles_titleId_get_404_response.json
+    post:
```

Comment:
This `post` section should be moved from `/eholdings/titles/{title_id}` to `/eholdings/titles` as correctly declared in `descriptors/ModuleDescriptor-template.json`.